src/Sentry/Internal/Enricher.cs

```diff
@@ -90,6 +91,12 @@ public void Apply(IEventLike eventLike)
                 eventLike.User.IpAddress ??= "{{auto}}";
             }
 
+            //Apply App startup time
+            eventLike.Contexts.App.StartTime ??= _options.StartupTime;
```

Comment:
Odd that this value is in the options. Can we have this is some internal type instead?